IPyth.sol

```diff
@@ -32,38 +32,58 @@ interface IPyth {
     /// @param fee Amount of paid fee for updating the prices.
     event UpdatePriceFeeds(address indexed sender, uint batchCount, uint fee);
 
-    /// @notice Returns the current price and confidence interval.
-    /// @dev Reverts if the current price is not available.
-    /// @param id The Pyth Price Feed ID of which to fetch the current price and confidence interval.
+    /// @notice Returns the price and confidence interval.
+    /// @dev Reverts if the price is not available.
```

Comment:
Reverts if the price has not been updated within the last `getValidTimePeriod()` seconds